extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -459,9 +461,14 @@ class MembershipsButtonEdit extends Component {
 								notices={ notices }
 							>
 								<div className="components-placeholder__instructions">
-									<p>{ __( 'Select payment plan:', 'jetpack' ) }</p>
+									<p>
+										{ __(
+											'To use this block, select a previously created payment plan.',
+											'jetpack'
+										) }
+									</p>
 									{ this.renderMembershipAmounts() }
-									<p>{ __( 'Or add another Recurring Payments plan:', 'jetpack' ) }</p>
+									<p>{ __( 'Or a new one.', 'jetpack' ) }</p>
 									{ this.renderAddMembershipAmount() }
```

Comment:
Since you've added a new parameter, what do you think about being explicit in each use of the function and always specify `true` or `false`? Wouldn't that make this a bit more readable?

Reply:
Oh, yeah, of course!
Fixed!